src/reducers/onboarding-status.reducer.js

```diff
@@ -48,18 +47,6 @@ export default (state: State = initialState, action: Action) => {
       return state === 'introducing-sidebar' ? 'done' : state;
     }
 
-    case REFRESH_PROJECTS: {
-      // If the disk reveals that this user already has guppy projects, we
-      // should assume that the onboarding has been completed previously.
-      // This can happen when Guppy has to rebuild the local storage from
-      // disk (which might happen after the app updates?)
-      if (Object.keys(action.projects).length) {
-        return 'done';
-      }
-
-      return state;
-    }
```

Comment:
We no longer support rebuilding the state from disk, so this isn't a concern anymore :)